client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -124,14 +126,17 @@ export default function CompositeCheckout( {
 	isComingFromUpsell?: boolean;
 	isLoggedOutCart?: boolean;
 	isNoSiteCart?: boolean;
-	isInEditor?: boolean;
+	isInModal?: boolean;
 	infoMessage?: JSX.Element;
 	onAfterPaymentComplete?: () => void;
-	isFocusedLaunch?: boolean;
+	disabledThankYouPage?: boolean;
 	isJetpackCheckout?: boolean;
 	jetpackSiteSlug?: string;
 	jetpackPurchaseToken?: string;
 	isUserComingFromLoginForm?: boolean;
+	customizedPreviousPath?: string;
+	/** Customized cancel url for PayPal */
+	customizedCancelUrl?: string;
```

Comment:
What was the purpose of this? It looks like it's being set to `window.location.href` which is the same value that's already used by the PayPal processor (although there's currently a bug where that URL loses the query string which will be fixed by https://github.com/Automattic/wp-calypso/pull/60476).

Reply:
Since I believe that this was probably added only to help with the bug also fixed by https://github.com/Automattic/wp-calypso/pull/60476, I've removed this variable entirely in that PR. Please let me know if I'm wrong!

Reply:
Yes, this is for keeping the query string in the cancel URL as I'm not sure is there any concern that we don't keep the query string. So, I think #60476 fix this issue and `customizedCancelUrl` can be removed 👍